pints/tests/test_log_priors.py

```diff
@@ -604,7 +604,13 @@ def test_multivariate_normal_prior(self):
         # Basic test
         covariance = [[1]]
         p = pints.MultivariateGaussianLogPrior(mean, covariance)
-        self.assertEqual(p([0]), -0.5 * np.log(2 * np.pi))
+        x = [0]
+        y = p(x)
+        self.assertEqual(y, -0.5 * np.log(2 * np.pi))
+        y1, dy = p.evaluateS1(x)
```

Comment:
Could we move this check down to the #test sensitivities bit? It's ok if that means recalculating x and y instead of storing them

Reply:
Done